### ISSUE TITLE
create an Apns2  app without persisting the certificate

### ISSUE THREAD

**Issue by csaurav** [rpush/rpush#652]
Hi, 

We have a requirement which doesn't allow us to persist the certificate in the db. We want to read the certificate value from a environment variable before sending the push notification. 

I didn't find any option of creating the Apns2 app without the certificate. Is their any way around ?

**Comment by benlangfeld** [comment]
I agree that this doesn't appear to be possible and would require a modification to rpush to allow it.
